projects/wm-core/src/tab-detail/tab-detail.component.html

```diff
@@ -29,81 +29,109 @@
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value">{{ distance}}km</ion-note>
-  </ion-item>
-
-  <ion-item *ngIf="properties?.duration_forward as duration_forward">
-    <i class="icon-outline-duration webmapp-pageroute-tabdetail-icon"> </i>
-    <ion-label start class="webmapp-pageroute-tabdetail-label">
-      {{ (properties?.roundtrip ? "duration":"duration_forward") | wmtrans }}
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value"
-      >{{duration_forward| duration }}
-    </ion-note>
-  </ion-item>
-
-  <ion-item
-    *ngIf="properties?.roundtrip === false && properties?.duration_backward as duration_backward"
-  >
-    <i class="icon-outline-duration webmapp-pageroute-tabdetail-icon"> </i>
-    <ion-label start class="webmapp-pageroute-tabdetail-label"
-      >{{ "duration_backward" | wmtrans }}
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value">
-      {{ duration_backward| duration }}
-    </ion-note>
-  </ion-item>
-
-  <ion-item *ngIf="properties?.ascent as ascent">
-    <i class="icon-outline-dislivello-positivo webmapp-pageroute-tabdetail-icon"> </i>
-    <ion-label start class="webmapp-pageroute-tabdetail-label">
-      {{"ascent" | wmtrans}}
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value"> {{ ascent }}m </ion-note>
-  </ion-item>
-
-  <ion-item *ngIf="properties?.roundtrip === false &&  properties?.descent as descent">
-    <i class="icon-outline-dislivello-negativo webmapp-pageroute-tabdetail-icon"> </i>
-    <ion-label start class="webmapp-pageroute-tabdetail-label">
-      {{ "descent" | wmtrans }}
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value"> {{ descent }}m</ion-note>
-  </ion-item>
-
-  <ion-item *ngIf="properties?.ele_from as ele_from">
-    <i class="icon-outline-plus webmapp-pageroute-tabdetail-icon"> </i>
-    <ion-label start class="webmapp-pageroute-tabdetail-label">
-      {{ "ele_from" | wmtrans }}
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value">{{ ele_from }}m</ion-note>
-  </ion-item>
-
-  <ion-item *ngIf="properties?.roundtrip === false && properties?.ele_to as ele_to">
-    <i class="icon-outline-minus webmapp-pageroute-tabdetail-icon"> </i>
-    <ion-label start class="webmapp-pageroute-tabdetail-label">
-      {{ "ele_to" | wmtrans }}
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value">{{ ele_to }}m</ion-note>
-  </ion-item>
-
-  <ion-item *ngIf="properties.ele_min as ele_min">
-    <i class="icon-outline-minus webmapp-pageroute-tabdetail-icon"> </i>
-    <ion-label start class="webmapp-pageroute-tabdetail-label">
-      {{ "ele_min" | wmtrans }}
-    </ion-label>
-    <ion-note slot="end" class="webmapp-pageroute-tabdetail-value">{{ ele_min }}m</ion-note>
-  </ion-item>
+  <ng-container *ngIf="(confOptions$ | async)?.showDistance">
```

Comment:
NO!
fare un ng-container con *ngIf=confOptions$ | async as confOptions in maniera tale da fare solo UNA async
e poi procedere sulle altre